_data/t--/t---.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Tyler Smith
+institution: PPU
+profile_pic: t---.png
+quote: Well, I made it
+github_user: t--
```

Comment:
Incorrect GitHub username, please change `t--` to `t---`.

Reply:
Fixed @Mohammed-Shoaib